services/ui/app/test_ui_analytics.py

```
@@ -0,0 +1,158 @@
+            "engine": "central-analytics-v1",
+            "query_version": "runtime-v1",
+            "run_id": "run-1",
+            "rows": [{"equipment_id": "AHU-1", "run_hours": 0.17}],
+        },
+    }
+    with patch.object(central_client, "health_ok", return_value=True):
+        with patch.object(central_client, "analytics_post", return_value=fake) as post:
+            out = ui_analytics.fetch_runtime_analytics(frames, role_map)
+    assert out.get("ok") is True
+    assert out["analytics"]["rows"][0]["equipment_id"] == "AHU-1"
+    args, kwargs = post.call_args
```

Ruff (0.16.0)

[warning] Unpacked variable `kwargs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)